components/List/src/MDCSelfSizingStereoCell.m

```diff
@@ -243,11 +246,11 @@ - (UIFont *)defaultDetailLabelFont {
 }
 
 - (UIColor *)defaultTitleLabelTextColor {
-  return [UIColor colorWithWhite:0 alpha:[MDCTypography subheadFontOpacity]];
```

Comment:
Please remove MDCTypography dependencies in the BUILD file and podspec.

Reply:
The file still uses (and was already using at the time of the initial change that is being rolled-back) MDCTypograghy, e.g. MDCFontTextStyleTitle in -defaultTitleLabelFont.